modules/collect/src/main/java/com/opengamma/strata/collect/timeseries/DenseLocalDateDoubleTimeSeries.java

```diff
@@ -57,7 +62,7 @@
    * array for weekends and providing the different date
    * calculations for each case.
    */
-  static enum DenseTimeSeriesCalculation {
+  enum DenseTimeSeriesCalculation {
```

Comment:
Agreed, but it is better for the source to be explicit, then it doesn't rely on esoteric knowledge of the code reader.